### PR TITLE
Added accessibility statement page and exception for it in middleware

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: src/authentication/middleware/index.ts [hunk 1]
```diff
@@ -18,20 +18,22 @@ export default (req: Request, res: Response, next: NextFunction) => {
   }
 
   logger.debug("Check if user is signed in");
-  if (!req.chSession.isSignedIn()) {
+  if (!req.originalUrl.endsWith(pageURLs.ACCESSIBILITY_STATEMENT)) {
+    if (!req.chSession.isSignedIn()) {
```
Comment (ICoxCH) [inline src/authentication/middleware/index.ts:22]: could these two not combined into a single OR operation?
If (notSignedIn OR accessingAccessibilityStatement)...

Comment (ICoxCH) [reply src/authentication/middleware/index.ts:22]: Ignore this comment it wouldn't work.

File: src/test/authentication/middleware/index.spec.unit.ts [hunk 2]
```diff
@@ -41,6 +41,13 @@ describe("Authentication middleware", () => {
     expect(response.status).toEqual(302);
   });
 
+  it("should not redirect to signin if loading accessibility statement page", async () => {
+    const response = await request(app)
+      .get("/extensions/accessibility-statement")
+      .set("Referer", "/extensions");
+    expect(response.status).toEqual(200);
```
Comment (markpit) [inline src/test/authentication/middleware/index.spec.unit.ts:48]: need to check what page is returned, could be returning 200 but for a different page

File: views/accessibility-statement.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        respond to your complaint,
+        <a href="https://www.equalityadvisoryservice.com/">contact the Equality Advisory and Support Service (EASS)</a>.</p>
+
+      <h2 class="govuk-heading-m">Technical information about this service’s accessibility</h2>
+      <p>Companies House is committed to making its website accessible in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.</p>
+      <p>This service is partially compliant with the
+        <a href="https://www.w3.org/TR/WCAG21/">Web Content Accessibility Guidelines version 2.1</a>
+        AA standard, due to the non-compliances listed below.</p>
+      <p>A page has an illogical heading structure. This fails WCAG 2.4.10 success criterion (section headings). We plan to fix this by April 2020.</p>
+      <p>Some links are out of context. This fails WCAG 2.4.9 success criterion (link purpose). We plan to fix this by April 2020.</p>
```
Comment (markpit) [inline views/accessibility-statement.html:64]: should this be April?  it's april now :)

File: views/accessibility-statement.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      <p>A links is present which does not indicate it will open in a new window. This fails WCAG 2.4.9 success criterion (link purpose). We plan to fix this by April 2020.</p>
+      <p>A timeout is present without indication to users. This fails WCAG 2.2.6 success criterion (timeouts). We plan to fix this by April 2020.</p>
+
+      <h2 class="govuk-heading-m">How we tested this service</h2>
+      <p>We regularly carry out internal tests for compliance with the Web Content Accessibility Guidelines V2.1 level A and level AA using a variety of manual and automated testing tools.</p>
+      <p>This service was last (externally) audited on 5 November 2018. The audit was carried out by the
```
Comment (markpit) [inline views/accessibility-statement.html:70]: is 2018 the date chs was auditted or extensions? extensions went live 2019.

File: views/accessibility-statement.html [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+            <a href="https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction#meeting-government-accessibility-requirements">government accessibility requirements</a>
+          </li>
+          <li>are developing
+            <a href="https://gds.blog.gov.uk/2018/06/20/creating-the-uk-governments-accessibility-empathy-lab/">an assistive technology lab</a>
+            in our Cardiff office to improve the testing of our services and raise awareness of accessibility</li>
+
+      <p>
+        <a href="https://companieshouse.blog.gov.uk/category/user-research/">Read our blogs about usability</a>
+        to find out more about our user research work.</p>
+      <p>This statement was prepared on 4 February 2020. It was last updated on 4 February 2020.</p>
```
Comment (markpit) [inline views/accessibility-statement.html:94]: another date to check

File: src/test/authentication/middleware/index.spec.unit.ts (hunk 2 above)
Comment (SVarallo) [reply src/test/authentication/middleware/index.spec.unit.ts:48]: Added a check for the page title

File: views/accessibility-statement.html (hunk 4 above)
Comment (SVarallo) [reply views/accessibility-statement.html]: Updated to 2019

File: views/accessibility-statement.html (hunk 5 above)
Comment (SVarallo) [reply views/accessibility-statement.html]: Changed the last updated date to today

File: views/accessibility-statement.html (hunk 3 above)
Comment (SVarallo) [reply views/accessibility-statement.html]: Updated to June

**Pull request merged**